packages/components/src/ListingDetailPhaseCard/ChallengeRevealVoteCard.tsx

```diff
@@ -42,14 +44,37 @@ export class ChallengeRevealVoteCard extends React.Component<
           <MetaItemValue>{this.props.stake}</MetaItemValue>
           <MetaItemLabel>Stake</MetaItemLabel>
         </StyledListingDetailPhaseCardSection>
-        <StyledListingDetailPhaseCardSection>
-          <RevealVote
-            salt={this.props.salt}
-            onInputChange={this.props.onInputChange}
-            transactions={this.props.transactions}
-          />
-        </StyledListingDetailPhaseCardSection>
+        <StyledListingDetailPhaseCardSection>{this.renderCommitVote()}</StyledListingDetailPhaseCardSection>
       </StyledListingDetailPhaseCardContainer>
     );
   }
+
+  private renderCommitVote = (): JSX.Element => {
```

Comment:
something about this function's name feels wrong.  should this be "renderRevealVote"